scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -450,20 +460,22 @@ task oncotate_m2 {
               ln -s oncotator_v1_ds_April052016 onco_dbdir
           fi
 
-
-        /root/oncotator_venv/bin/oncotator --db-dir onco_dbdir/ -c $HOME/tx_exact_uniprot_matches.AKT1_CRLF2_FGFR1.txt  \
-            -v ${m2_vcf} ${entity_id}.oncotated.vcf hg19 -i VCF -o VCF --infer-onps --collapse-number-annotations --log_name oncotator.log
+        ${default="/root/oncotator_venv/bin/oncotator" oncotator_exe} --db-dir onco_dbdir/ -c $HOME/tx_exact_uniprot_matches.AKT1_CRLF2_FGFR1.txt  \
+            -v ${m2_vcf} ${entity_id}.maf.annotated hg19 -i VCF -o TCGAMAF --skip-no-alt --infer-onps --collapse-number-annotations --log_name oncotator.log \
+            -a Center:${default="Unknown" sequencing_center} \
+            -a source:${default="Unknown" sequence_source} \
+            ${"--default_config " + default_config_file}
     }
 
     runtime {
         docker: "${oncotator_docker}"
-        memory: "5 GB"
-        bootDiskSizeGb: 10
-        disks: "local-disk 150 SSD"
+        memory: "3 GB"
+        bootDiskSizeGb: 12
+        disks: "local-disk 100 HDD"
         preemptible: "${preemptible_attempts}"
     }
 
     output {
-        File oncotated_m2_vcf="${entity_id}.oncotated.vcf"
+        File oncotated_m2_maf="${entity_id}.maf.annotated"
```

Comment:
not sure about this (not familiar with MAF format), but I would, intuitively, expect a file named "*.annotated.maf".